hathor/builder/builder.py

```diff
@@ -473,24 +473,20 @@ def _get_or_create_bit_signaling_service(self) -> BitSignalingService:
     def _get_or_create_verification_service(self) -> VerificationService:
         if self._verification_service is None:
             verifiers = self._get_or_create_vertex_verifiers()
-            self._verification_service = VerificationService(verifiers=verifiers)
+            feature_service = self._get_or_create_feature_service()
+            self._verification_service = VerificationService(verifiers=verifiers, feature_service=feature_service)
 
         return self._verification_service
 
     def _get_or_create_vertex_verifiers(self) -> VertexVerifiers:
```

Comment:
Why not simply verify whether it is enabled or not here?

Reply:
This has been removed from this PR.